app/config/variables.php

```diff
@@ -1119,6 +1119,15 @@
         'category' => 'Assistant',
         'description' => '',
         'variables' => [
+            [
+                'name' => '_APP_ASSISTANT_HOST',
+                'description' => 'Host and port of the assistant container.',
+                'introduction' => '1.4.0',
```

Comment:
```suggestion
                'introduction' => '1.4.0',
```
this version probably need to be updated before a release 👍🏻